src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/integration/sentencing/SentencingMigrationIntTest.kt

```diff
@@ -95,7 +95,7 @@ class SentencingMigrationIntTest : SqsIntegrationTestBase() {
       mappingApi.stubAllNomisSentencingAdjustmentsMappingNotFound()
       mappingApi.stubSentenceAdjustmentMappingCreate()
 
-      sentencingApi.stubCreateSentencingAdjustment()
+      sentencingApi.stubCreateSentencingAdjustmentForMigration()
       mappingApi.stubSentenceAdjustmentMappingByMigrationId(count = 86)
```

Comment:
We expect 2 endpoint in Digital Wallet that more or less the same thing, one for migration for synchronisation